server/Logic/WaiterOrder.ts

```diff
@@ -32,38 +32,24 @@ export async function updateWaiterLocation(
 }
 
 export async function assignWaiter(
```

Comment:
We should check if some of the waiters is already assigned to the requested order.
If some of them are assigned then we can return an error or proced normally but we dont want to assign again.
I think assigning a waiter twice will result with them getting the same notifications twice.